core-compose/src/main/java/com/squareup/workflow/ui/compose/internal/ViewFactories.kt

```diff
@@ -93,8 +98,15 @@ import kotlin.properties.Delegates.observable
   val wrappedEnvironment = remember(viewEnvironment) {
     viewEnvironment + (ParentComposition to parentComposition)
   }
-
-  HostView(viewFactory = viewFactory, update = Pair(rendering, wrappedEnvironment))
+  val context = ContextAmbient.current
+  val view = remember(context) { HostView(context) }.apply {
+    this.viewFactory = viewFactory
+    update = Pair(rendering, wrappedEnvironment)
+  }
+  AndroidView(
+      view = view,
+      modifier = Modifier.testTag(LegacyAndroidViewTestTag)
+  )
```

Comment:
I think we can use the new `emitView` API here instead. Gonna try it locally and report back.

Reply:
Yep, it works! Let me know if you want me to post the code I ended up with.